scripts/setup-db-user.py

```diff
@@ -51,14 +51,20 @@ def main(role, create_role, print_variables):
     passw = passw.strip()
     if print_variables:
         prefix = role.upper()
-        print("{0}_POSTGRES_DB={1}; export {0}_POSTGRES_DB".format(prefix, role))  # noqa
-        print("{0}_POSTGRES_USER={1}; export {0}_POSTGRES_USER".format(prefix, role))  # noqa
-        print("{0}_POSTGRES_PASSWORD={1}; export {0}_POSTGRES_PASSWORD".format(prefix, passw))  # noqa
+        print("{0}_POSTGRES_DB={1}; export {0}_POSTGRES_DB".format(  # noqa
+            prefix, role))
+        print("{0}_POSTGRES_USER={1}; export {0}_POSTGRES_USER".format(  # noqa
```

Comment:
Yeah good point, I know that problem! A case of DRY vs. readability, I guess.

In this case this happens twice, once for `clims` and once for `test_clims`, so it's this or copy the code. It might be good to document it somewhere instead, e.g. in a comment above this. I'll keep that in mind!

Reply:
After thinking about it, I think that in this case readability (or searchability) should weigh more than DRY, but I think I'll just fix it next time I have to work on this, if that's cool.

Reply:
Thanks for the feedback! I think it would be cool to merge this now and fix this soon (TM).